dockerignore

```diff
@@ -10,3 +10,38 @@ env.j2
 ### docker files ###  
```

Comment:
I don't know if I spotted this before, but typically this file would be `.dockerignore` rather than `dockerignore`